<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightDialog</class>
 <widget class="QDialog" name="LightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="4">
    <widget class="QDoubleSpinBox" name="direction_y">
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_7">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>direction vector</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>diffuse</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QDoubleSpinBox" name="direction_x">
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QComboBox" name="curve_selection">
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>cos smooth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cos absolute value</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>linear half</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>linear full</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="8">
    <widget class="QDoubleSpinBox" name="viewer_height">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="label_6">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>viewer height</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QDoubleSpinBox" name="phong_exponent">
     <property name="minimum">
      <double>-10.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Phong exponent</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>specular</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QFrame" name="frame_ambient">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QFrame" name="frame_curve">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QFrame" name="frame_specular">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QFrame" name="frame_diffuse">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pick_specular">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>pick</string>
     </property>
    </widget>
   </item>
   <item row="7" column="8">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pick_diffuse">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>pick</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pick_ambient">
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>pick</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ambient</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>reflection curve</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LightDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LightDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
